crypto/fipsmodule/ml_kem/ml_kem_ref/kem.c

```diff
@@ -9,6 +9,21 @@
 #include "symmetric.h"
 #include "openssl/rand.h"
 
+// FIPS 203. Pair-wise Consistency Test (PCT) required per [FIPS 140-3 IG](https://csrc.nist.gov/csrc/media/Projects/cryptographic-module-validation-program/documents/fips%20140-3/FIPS%20140-3%20IG.pdf):
+// The PCT consists of applying the encapsulation key to encapsulate a shared
+// secret leading to ciphertext, and then applying decapsulation key to
+// retrieve the same shared secret. Returns 0 if the PCT passes, 1 otherwise.
+static int keygen_pct(ml_kem_params *params, const uint8_t *ek, const uint8_t *dk) {
```

Comment:
NP: Would it be more clear to name  `ek` as `pk` (public key), and `dk` as `sk` (secret key)? These are the names/terms used by the caller below.

Reply:
we are moving towards using `ek` and `dk` to align with FIPS 203 naming